data/hai/hai missions.txt

```diff
@@ -93,7 +111,7 @@ mission "First Contact: Unfettered"
 				`	"What do you mean, 'true Hai'?"`
 					goto true
 				`	"Why are you at war with everyone else?"`
-					goto everyone
+					goto evelse
```

Comment:
Labels should still be words. To avoid duplicate labels, I'd do `everyone1` and `everyone2` instead of `everyone` and `evelse`, or perhaps `everyone` for one of the labels and `war` for another.

Reply:
Yeah, I agree. I realized afterwards that this (and one of the Unfettered lines) should be changed a little, I figured it was best to wait until a second pass, instead of keep spamming changes.

Reply:
Actually, it was the Unfettered first version of this that I intended to change, so the original will revert to Everyone, and the new version will just be the player asking if they are at war.